test/powershell/Common/TestHostCS.psm1

```diff
@@ -191,7 +196,7 @@ namespace TestHost
 
         public override void WriteLine(string value)
         {
-            Streams.ConsoleOutput.Add(value);
+            Streams.ConsoleOutput.Add("::"+value+":NewLine");

```

Comment:
I see your point now. It sounds over complex and useful at the same time 😄 
I'm OK to have this change in TestHostCS.psm1, but could you please put more comments there for the `Write/WriteLine` APIs so that people can a bit easier to understand the behavior?